internal/comment/bitbucket.go

```diff
@@ -794,3 +794,7 @@ func (h *bitbucketServerPRHandler) fetchServerComment(commentURL string) (*bitbu
 
 	return &resData, nil
 }
+
+func bitbucketMarkdownTag(tag string) string {
+	return fmt.Sprintf("\n\n*(%s)*", tag)
```

Comment:
I wonder if we should include these two empty lines, it's more formatting to me, `*(infracost-comment)*` should be enough to detect. What do you think?